front_end/ui/legacy/RemoteDebuggingTerminatedScreen.ts

```diff
@@ -27,12 +27,24 @@ const UIStrings = {
    * device back in a state where it can be inspected, before DevTools can reconnect to it.
    */
   reconnectWhenReadyByReopening: 'Reconnect when ready by reopening DevTools.',
```

Comment:
I think we can also make this slightly more clear.

```suggestion
  reconnectWhenReadyByReopening: 'Reconnect when ready (will reload DevTools).',
```